.github/workflows/maven.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Can you modify this to `main` so the tests run?

Reply:
> Can you modify this to `main` so the tests run?

Yikes...sorry that I missed that. It is fixed now.